CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @inrupt/sdk-owners @inrupt/apps-owners
```

Comment:
Should app-owners be here too?

Reply:
Note, that's a type, fixed in 17dbb67b2d152e08c906805deed3b177777ba356